cmd/multi_cmd.go

```diff
@@ -37,5 +37,5 @@ func multi(config *conf.GlobalConfiguration) {
 
 	models.RunHooks(bgDB, logrus.WithField("component", "hooks"))
 
-	api.ListenAndServe(l)
+	logrus.WithError(api.ListenAndServe(l)).Fatal("Error listening")
```

Comment:
The docs are correct. I looked at the code and in the case of a clean shutdown, it returns `ErrServerClosed` which is a string `http: Server closed`.